checkstyle-configuration.xml

```diff
@@ -109,13 +122,17 @@
             <property name="option" value="EOL" />
         </module>
 
+        <module name="CommentsIndentation" />
+
         <!-- Modifier Checks                                    -->
         <!-- See http://checkstyle.sf.net/config_modifiers.html -->
         <module name="ModifierOrder" />
         <module name="RedundantModifier" />
 
         <module name="LeftCurly">
-            <property name="option" value="nl" />
+            <property name="option" value="nl"/>
+            <!--TODO: Remove this, to include the LAMBDA token-->
+            <property name="tokens" value="INTERFACE_DEF, CLASS_DEF, ANNOTATION_DEF, ENUM_DEF, CTOR_DEF, METHOD_DEF, ENUM_CONSTANT_DEF, LITERAL_WHILE, LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_SYNCHRONIZED, LITERAL_SWITCH, LITERAL_DO, LITERAL_IF, LITERAL_ELSE, LITERAL_FOR, STATIC_INIT, OBJBLOCK"/>
```

Comment:
Older versions of CheckStyle didn't enforce any convention for lambda curly braces, and so our codebase has a lot of inconsistencies. If we choose to fix them, I'd want to do so in a separate commit. For now, I listed all tokens as eligible for checking except for lambdas.